Source/Common/ContextJsonConverter.cs

```diff
@@ -11,46 +11,34 @@ namespace Schema.NET
     public class ContextJsonConverter : JsonConverter<JsonLdContext>
     {
         /// <inheritdoc />
-        public override JsonLdContext ReadJson(JsonReader reader, Type objectType, JsonLdContext? existingValue, bool hasExistingValue, JsonSerializer serializer)
+        public override JsonLdContext Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
         {
-            if (reader is null)
+            if (typeToConvert is null)
             {
-                throw new ArgumentNullException(nameof(reader));
+                throw new ArgumentNullException(nameof(typeToConvert));
             }
 
-            if (objectType is null)
-            {
-                throw new ArgumentNullException(nameof(objectType));
-            }
-
-            if (hasExistingValue && existingValue is null)
-            {
-                throw new ArgumentNullException(nameof(existingValue));
-            }
-
-            if (serializer is null)
-            {
-                throw new ArgumentNullException(nameof(serializer));
-            }
-
-            var context = hasExistingValue ? existingValue! : new JsonLdContext();
+            var context = new JsonLdContext();
 
-            string? name;
-            string? language;
-            if (reader.TokenType == JsonToken.String)
+            string? name = null;
+            string? language = null;
+            if (reader.TokenType == JsonTokenType.String)
             {
-                name = (string?)reader.Value;
-                language = null;
+                name = reader.GetString();
             }
             else
             {
-                var o = JObject.Load(reader);
+                var o = JsonDocument.ParseValue(ref reader);
 
-                var nameProperty = o.Property("name", StringComparison.OrdinalIgnoreCase);
-                name = nameProperty?.Value?.ToString() ?? "https://schema.org";
+                if (o.RootElement.TryGetProperty("name", out var nameElement))
+                {
+                    name = nameElement.GetString() ?? "http://schema.org";
+                }
 
-                var languageProperty = o.Property("@language", StringComparison.OrdinalIgnoreCase);
-                language = languageProperty?.Value?.ToString();
+                if (o.RootElement.TryGetProperty("@language", out var languageElement))
+                {
+                    language = languageElement.GetString();
+                }
             }
 
 #pragma warning disable CA1062 // Validate arguments of public methods
```

Comment:
We can now remove this warning suppression. The weird parameters of JSON.NET are gone.